lib/Util/Notices/DiscountsAnnouncement.php

```diff
@@ -0,0 +1,44 @@
+<?php
+namespace MailPoet\Util\Notices;
+
+use MailPoet\Models\Setting;
+use MailPoet\Util\Helpers;
+use MailPoet\WP\Notice as WPNotice;
+
+class DiscountsAnnouncement {
+
+  const OPTION_NAME = 'mailpoet_display_discounts_announcement';
```

Comment:
Isn't this too general? What if we add and more discount announcements in the future. Maybe we can add some date suffix so that we can distinguish them in the future. Maybe `mailpoet_display_discounts_announcement_q4_2018` ? What do you think?